src/main/java/frc/utils/ctre/BusChain.java

```diff
@@ -4,59 +4,74 @@
 import edu.wpi.first.wpilibj.Timer;
 import frc.robot.constants.LogPaths;
 import org.littletonrobotics.junction.Logger;
+import com.ctre.phoenix6.CANBus.CANBusStatus;
 
 public enum BusChain {
 
 	ROBORIO("rio"),
 	CANIVORE("CANivore");
 
+	private static final double MAX_CAN_UTILIZATION_DECIMAL_PERCENT = 0.6;
+	private static final double MAX_RECEIVE_ERRORS = 0;
+	private static final double MAX_TRANSMIT_ERRORS = 0;
```

Comment:
add the word permitted of somethong like that, because its not max but the number that is problematic, like utilization can be 1 but its a problem when it gets above 0.6

Reply:
ok